components/style/build.rs

```diff
@@ -8,12 +8,32 @@ use std::io::Write;
 use std::path::Path;
 use std::process::{Command, Stdio, exit};
 
+#[cfg(windows)]
+fn find_python() -> String {
+    if Command::new("python27.exe").arg("--version").output().is_ok() {
+        return "python27.exe".to_owned();
+    }
 
-fn main() {
-    let python = if Command::new("python2.7").arg("--version").output().unwrap().status.success() {
+    if Command::new("python.exe").arg("--version").output().is_ok() {
+        return "python.exe".to_owned();
+    }
+
+    panic!("Can't find python (tried python27.exe and python.exe)! Try fixing PATH or setting the PYTHON env var");
+}
+
+#[cfg(not(windows))]
+fn find_python() -> String {
+    if Command::new("python2.7").arg("--version").output().unwrap().status.success() {
         "python2.7"
     } else {
         "python"
+    }.to_owned()
+}
+
+fn main() {
+    let python = match env::var("PYTHON") {
```

Comment:
Do we really need this here? It seems like if the user doesn't have Python 2.7 installed, they'll find out by `mach` anyways [here](https://github.com/servo/servo/blob/9fca41c7a7498cb6e27b914c45290dc4639d6f8b/python/mach_bootstrap.py#L90-L92).